src/wp-content/themes/twentyfourteen/functions.php

```diff
@@ -351,8 +351,7 @@ function twentyfourteen_scripts() {
 	wp_enqueue_style( 'twentyfourteen-block-style', get_template_directory_uri() . '/css/blocks.css', array( 'twentyfourteen-style' ), '20230206' );
 
 	// Load the Internet Explorer specific stylesheet.
-	wp_enqueue_style( 'twentyfourteen-ie', get_template_directory_uri() . '/css/ie.css', array( 'twentyfourteen-style' ), '20140711' );
-	wp_style_add_data( 'twentyfourteen-ie', 'conditional', 'lt IE 9' );
```

Comment:
The conditional data should stay.

Reply:
Restored `wp_style_add_data` and updated `Load` with `Register` in the comment for the `wp_register_style`